includes/admin/payments/view-refund.php

```diff
@@ -18,6 +18,11 @@
  * @since 3.0
  */
 function edd_view_refund_page_content() {
+
+	wp_enqueue_script( 'edd-admin-orders' );
+	// Enqueued for backwards compatibility. Empty file.
+	wp_enqueue_script( 'edd-admin-payments' );
+
```

Comment:
This can be moved under the potential `wp_die()` calls so they don't get enqueued until we know we have something to display.